CheckpointPlugin.cpp

```diff
@@ -489,7 +517,7 @@ void CheckpointPlugin::doCheckpoint(std::vector<std::string> command) {
 		curCheckpoint = checkpoints.size();
 		checkpoints.push_back(latest);
 		saveCheckpointFile();
-		loadGameState(latest);
+		// loadGameState(latest);
```

Comment:
removing this line lets you set multiple checkpoints from the same play without erasing your history :)

Reply:
hmmm sure i'll put it in it's own PR for sure! for that other PR, could we have it as an option? i really like this feature since i can do one play and then save the key moments to practice each part 😍

Reply:
I'm fine with an option for this, but we should make sure something is displayed when you save the checkpoint, and a second push of doCheckpoint like this should probably delete the new checkpoint.  I.e.:

* Rewinding...
* `cpt_do_checkpoint` - Message appears: "Checkpoint saved.  Press again to remove."
* `cpt_do_checkpoint` again - Message appears: "Checkpoint removed."